Content revision

PsychSec opened the pull request:
This pull request contains multiple changes on the e-tray/inbox to update the branch for our current study

== Content ==
Added the instructions we originally developed for our e-tray exercise.
Added UdS version of consent.
Some minor css and styling changes.

== Functionality  ==
Added the front-end function to create emails (the data is not stored at the backend yet, as I did not want to mess with the firebase database) - for now the data is just logged at the console
Added the --openssl-legacy-provider flag for convenience (when running with newer npm versions)

Cheers,
Stefan

PsychSec committed:
Instruction page and minor style-changes

PsychSec committed:
Revision of consent page - UdS standards

PsychSec committed:
Added create email function

ucbtszh reviewed (changes requested):
Line 306 in Inbox: do you want users to be able to create a new e-mail after they have processed e-mail(s) in the inbox or while they are viewing another e-mail? If so, you don't necessarily need "emlViewIndex == void(0)". I would also suggest to use "emlViewIndex == null" as a best practice, especially since emlViewIndex equals 'null' and not void(0) (which means 'undefined') in the current implementation.

Minor comment to consistently use either "e-mail" or "email" in wording. 

Layout changes look fine to me.

PsychSec committed:
Email Wording + create email fix

PsychSec committed:
Fixed create email (for real this time) and reverted skip captcha from previous commit

PsychSec commented:
Thank you for the review! I made the following changes:

- Replaced "e-mail" with "email."
- You were absolutely right about "emlViewIndex == void(0)." I removed it and instead added "emlViewIndex = null" to the create button. This way, participants can access the create email field at any given time, just like in a real mail client.
- I reverted a change I had previously made, where I hard-coded a bypass for the captcha. It was useful for testing, but it's probably better not to have it in the repository.
